resallocserver/logic.py

```diff
@@ -105,7 +105,7 @@ def stats(self):
         items = {}
         items['on']     = self.on().all()
         items['up']     = [x for x in items['on'] if x.state  == RState.UP]
-        items['ready']  = [x for x in items['up'] if x.ticket_id is None]
+        items['ready']  = [x for x in items['up'] if x.ticket_id is None and x.sandbox is None]
```

Comment:
I don't think we want to change the semantics of the "ready" flag.  Resource is ready when it can be taken (assigned to ticket).  We should rather fix the resource allocation mechanism so it better calculates those sandboxed resources.  I'd probably rather see something like 'resource.released > 0' check as the sandboxing is/could be optional.